core/src/dird/ua_run.cc

```diff
@@ -723,18 +723,24 @@ int ModifyJobParameters(UaContext* ua, JobControlRecord* jcr, RunContext& rc)
         } else if (jcr->is_JobType(JT_RESTORE)) { /* Where */
           if (GetCmd(ua, T_("Please enter the full path prefix for restore (/ "
                             "for none): "))) {
-            if (jcr->RegexWhere) { /* cannot use regexwhere and where */
-              free(jcr->RegexWhere);
-              jcr->RegexWhere = NULL;
-            }
-            if (jcr->where) {
-              free(jcr->where);
-              jcr->where = NULL;
-            }
-            if (IsPathSeparator(ua->cmd[0]) && ua->cmd[1] == '\0') {
-              ua->cmd[0] = 0;
+            if (!ua->AclAccessOk(Where_ACL, ua->cmd, true)) {
```

Comment:
This should probably also happen in case 10 (File Relocation) where regexwhere is set instead.

Reply:
sure.